src/btree/bt_read.c

```diff
@@ -194,8 +200,25 @@ __las_page_instantiate(WT_SESSION_IMPL *session, WT_REF *ref)
     locked = false;
 
     /* Check whether the disk image contains all the newest versions of the page. */
-    if (ref->page_las->min_skipped_ts == WT_TS_MAX)
-        return (__instantiate_birthmarks(session, ref));
+    if (page_las->min_skipped_ts == WT_TS_MAX) {
+        WT_RET(__instantiate_birthmarks(session, ref));
+
+        FLD_SET(page->modify->restore_state, WT_PAGE_RS_LOOKASIDE);
```

Comment:
The disk image that is written already contains the newest version of the page, so there is nothing to be done by the next checkpoint, because of this reason I didn't set that here. Even setting it with `WT_TXN_FIRST` doesn't matter with next checkpoint, it is better to do it. Opinions?